src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -762,6 +722,46 @@ bool KivaManager::setupKivaInstances()
 			// Remaining exposed perimeter will be alloted to each instance as appropriate
 			Real64 remainingExposedPerimeter = totalExposedPerimeter;
 
+			// Get combinations of wall constructions and wall heights -- each different
+			// combination gets its own Kiva instance. Combination map points each set
+			// of construction and wall height to the associated exposed perimeter and
+			// list of wall surface numbers.
+			std::map<std::pair<int, Real64>,WallGroup> combinationMap;
+
+			if ( wallSurfaces.size() != 0 ) {
+				for ( auto& wl : wallSurfaces ) {
+
+					auto&v = Surfaces( wl ).Vertex;
```

Comment:
Whitespace here, and for the time being, we should probably go ahead and pad the angle brackets in the map declaration a few lines up, just to keep it uniform with existing code.